products/securityscanner/api.yaml

```diff
@@ -0,0 +1,157 @@
+# Copyright 2019 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+--- !ruby/object:Api::Product
+name: SecurityScanner
+display_name: Cloud Security Scanner
+versions:
+  - !ruby/object:Api::Product::Version
+    name: beta
+    base_url: https://websecurityscanner.googleapis.com/v1beta/
+scopes:
+  - https://www.googleapis.com/auth/cloud-platform
+apis_required:
+  - !ruby/object:Api::Product::ApiReference
+    name: Web Security Scanner API
+    url: https://console.cloud.google.com/apis/library/websecurityscanner.googleapis.com/
+objects:
+  - !ruby/object:Api::Resource
+    name: ScanConfig
+    base_url: projects/{{project}}/scanConfigs
+    self_link: "{{name}}"
+    update_verb: :PATCH
+    update_mask: true
+    description: |
+      A ScanConfig resource contains the configurations to launch a scan.
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        output: true
+        description: |
+          A server defined name for this index. Format:
+          `projects/{{project}}/scanConfigs/{{server_generated_id}}`
+      - !ruby/object:Api::Type::String
+        name: displayName
+        required: true
+        description: |
+          The user provider display name of the ScanConfig.
+      - !ruby/object:Api::Type::Integer
+        name: maxQps
+        description: |
+          The maximum QPS during scanning. A valid value ranges from 5 to 20 inclusively.
+          If the field is unspecified or its value is set 0, server will default to 15.
```

Comment:
nit: its value is set to 0 (it's ok if our documentation doesn't match the one from the site exactly)

Reply:
Ah sorry let me clarify- the sentence should be "If the field is unspecified or its value is set to 0, the server will default to 15."

Reply:
Ah, yeah didn't catch that